polygon-digital-carbon/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "polygon-carbon",
-  "version": "1.6.7",
+  "name": "polygon-digital-carbon",
```

Comment:
this shouldn't cause any gha issue as we are pulling from the subgraph name from the folders, but probably something to watch out for.